packages/common/index.ts

```diff
@@ -0,0 +1,4 @@
+// -- Utils -------------------------------------------------------------------
```

Comment:
Note for follow up PR's, not this one, we should also probably start moving utils from core and ui here

Reply:
Definitely, let me note this. and see what we can move from there.